pom.xml

```diff
@@ -274,10 +274,15 @@
         </dependency>
         <dependency>
             <groupId>org.mockito</groupId>
-            <artifactId>mockito-inline</artifactId>
-            <version>4.6.1</version>
+            <artifactId>mockito-core</artifactId>
+            <version>5.5.0</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>net.bytebuddy</groupId>
+            <artifactId>byte-buddy</artifactId>
+            <version>1.14.8</version>
```

Comment:
While we're here can we update them to the latest version on maven?

Reply:
Initially left them the same as previous 2 service changes for consistency. Will retest on latest version as well.

Reply:
Yeah that makes sense, I'm not sure which way we want to go...